Arrays/numbers-filtering.js

```diff
@@ -0,0 +1,50 @@
+let numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+
+let evenNumbers = numbers.filter(e => e % 2 === 0
+
+)
+ console.log(evenNumbers);
+
+
+
+ //4. Give the below array of object:
+
+ const userScore = [  
+    {id: 1, name: 'Olarike', score: 34},  
+    {id: 2, name: 'Taiwo', score: 12},  
+    {id: 3, name: 'Kehinde', score: 27} ];
+
+
+// Find where user score above 12 and return the user info.
+
+let aboveTwelve = userScore.filter(e => e.score > 12)
+console.log(aboveTwelve)
+
+// Sort the array in descending order based on the user score.
+ userScore.sort(function (a,b) {
+     return b.score - a.score
+ })
+
+console.log(userScore)
+
+// Loop through the array after sort and return the each user. Note your position should start at 1.
+
+for (index = 1; index < userScore.length; index++) { 
+    console.log(userScore[index]); 
+} 
```

Comment:
@Beckyady-Kadet this will not return all user in the array

Reply:
Yes the loop does not return all users in the array